continuous-integration/styleci/configurations/general-php.md

```diff
@@ -0,0 +1,49 @@
+# StyleCI - General PHP
+
+The following is a recommended configuration for general PHP projects which may or may not use a framework. If you are
+using Laravel please use the alternative configuration [here](laravel.md). This configuration uses the `psr2` 
+preset with risky heuristics enabled. It is also tailored to suit our internal PHP & JavaScript guidelines.
+
+```yaml
+php:
+  preset: psr2
+  
+  risky: true
+
+  enabled:
+    - concat_with_spaces
+
+  disabled:
+    - short_array_syntax
+    - phpdoc_summary
+    - length_ordered_imports
+    - phpdoc_no_package
+    - concat_without_spaces
```

Comment:
I feel like we should leave this out (as default) until consensus is reached in #74.

Reply:
https://github.com/langleyfoxall/technologies/pull/76#discussion_r229276821